test/functionalTest/cases/0000_ngsild/ngsild_url_parse.test

```diff
@@ -439,12 +439,12 @@ Date: REGEX(.*)
 17. DELETE /ngsi-ld/v1/subscriptions/S1
 =======================================
 HTTP/1.1 404 Not Found
-Content-Length: 141
+Content-Length: 145
 Content-Type: application/json
 Date: REGEX(.*)
 
 {
-    "detail": "S1",
+    "detail": "urn:S1",
     "title": "The requested subscription has not been found. Check id",
```

Comment:
Once you change the error message, you'll have to change it in all failing functests as well - this file and perhaps some older file - run "ft -ld" and fix all before sending a new commit